demos/CMakeLists.txt

```diff
@@ -111,14 +108,15 @@ create_demo(distance-field distance-field.c)
 create_demo(distance-field-2 distance-field-2.c)
 create_demo(distance-field-3 distance-field-3.c)
 
+
 if(FONTCONFIG_FOUND)
    create_demo(markup markup.c)
    target_link_libraries(markup ${FONTCONFIG_LIBRARY})
 endif()
 
 if(ANT_TWEAK_BAR_FOUND)
     create_demo(atb-agg atb-agg.c)
-    target_link_libraries(atb-agg ${ANT_TWEAK_BAR_LIBRARY})
+    target_link_libraries(atb-agg ${ANT_TWEAK_BAR_LIBRARY} stdc++)
```

Comment:
No check for availability of std++ is made.

Probably the correct way to fix this is to set the language of the atb-agg target to c++

```
set_target_properties(atb-agg PROPERTIES LINKER_LANGUAGE CXX)
```

http://public.kitware.com/pipermail/cmake/2006-July/010380.html